Source/Content.swift

```diff
@@ -14,9 +18,10 @@ public class Content: NSObject {
   public private(set) var initialPosition: Position
   private let group = ConstraintGroup()
 
-  public init(view: UIView, position: Position) {
+  public init(view: UIView, position: Position, centered: Bool = true) {
```

Comment:
I think I will remove `true` from variable, because default value in initializer could be useful to write less code.